docs/reference/data-frames/apis/delete-transform.asciidoc

```diff
@@ -2,8 +2,10 @@
 [testenv="basic"]
 [[delete-data-frame-transform]]
 === Delete {dataframe-transforms} API
+
+[subs="attributes"]
 ++++
-<titleabbrev>Delete {dataframe-transforms}</titleabbrev>
+<titleabbrev>Delete data frame transforms</titleabbrev>
```

Comment:
I think you can revert this line now that you have `subs=attributes`.

Reply:
:doh: Sorry about that @nik9000. For some reason, I thought I changed both the titles and titleabbrevs. Looks like I only did the titles though.

Should be fixed up with 71460d7.